battle-service/src/main/java/com/adavydenko/tictactoe/battleservice/entities/Battle.java

```diff
@@ -0,0 +1,43 @@
+package com.adavydenko.tictactoe.battleservice.entities;
+
+import com.adavydenko.tictactoe.userservice.entities.User;
+import jakarta.persistence.Entity;
+import jakarta.persistence.Id;
+import jakarta.persistence.ManyToOne;
+import jakarta.persistence.OneToMany;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+import java.time.LocalDateTime;
+import java.util.List;
+
+@Entity
+@Getter
+@Setter
+@AllArgsConstructor
+@NoArgsConstructor
+public class Battle {
+    @Id
+    private String id;
+    private BattleStatus status;
+    @ManyToOne
+    private User playerOne; // player X
```

Comment:
playerX playerY

Reply:
What do you mean? I made it this way: a player one plays with X mark and player two with O

Reply:
I meant rename playrOne to playerX and playerTwo to playerY, to makr it obvious